### PR TITLE
Add Modal to docs site

### PR DESCRIPTION
**Pull request by unscsprt**
Addresses #471

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "is-absolute-url": "^2.1.0",
     "mdx-docs": "^1.0.0-9",
     "pcln-design-system": "^2.0.5",
```
Comment (sdalonzo) [inline docs/package.json]: n-b bump to latest?

**Comment by jes708** [comment]
Might it make sense to rename the Components section as Core Components, give the modal its own Modal Components section, and then provide docs for each export of pcln-modal (headers, etc)?

**Comment by dgiraldo313** [comment]
> Might it make sense to rename the Components section of the docs as Core Components, give the modal its own Modal Components section, and then provide docs for each export of pcln-modal (headers, etc)?

I think that's a great idea. I've seen confusion from some people thinking that the modal, popover and other standalone packages come with the DS core. I think separating these in the docs can help alleviate some of this confusion.

**Comment by unscsprt** [comment]
@dgiraldo313 @jes708 Good idea; created separate section for Modal components.

Screen Shot 2019-08-09 at 12 17 29 PM

**Pull request merged**